tests/e2e_tests/utils/commands.ts

```diff
@@ -288,3 +288,18 @@ export const compareImages = (expectedPath, actualPath, options = { threshold: 0
   const pass = usePercentage ? (numDiffPixels / (width * height)) * 100 < threshold : numDiffPixels < threshold;
   return { pass, numDiffPixels };
 };
+
+export const tagRelease = async (releaseName: string, tag: string, baseUrl: string, token: string) => {
+  const request = await axios.put(`${baseUrl}api/management/v2/deployments/deployments/releases/${releaseName}/tags`, [tag], {
```

Comment:
this URL is not defined anywhere?

Reply:
Well it is in the app code, but here the straight API calls should remain the exception - once I get to reuse the URL I would put it in a const 👌